frontend/components/tasks/affectiveAnnotation/offensive/OffensiveInput.vue

```diff
@@ -470,6 +453,16 @@ export default Vue.extend({
 .offensive-input {
     max-height: 400px;
     overflow-y: auto;
+
+    &.--bordered {
+        border: 2px solid #f0f0f0;
+        border-radius: 2px;  
```

Comment:
imo the color `#f0f0f0` can be made darker a bit more, maybe something like `#ddd`, because right now the border/segmentation between the input components are still hardly visible. 😅